engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -185,8 +185,9 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId,
       context.actorOf(actorProps, buildJobExecutionActorName(jobDescriptor)) ! ExecuteJobCommand
       stay() using mergeExecutionDiff(stateData, WorkflowExecutionDiff(Map(jobDescriptor.key -> ExecutionStatus.Running)))
     case Event(BackendJobPreparationFailed(jobKey, t), stateData) =>
-      log.error(s"Failed to start job $jobKey", t)
+      log.error(t, "Failed to start job {}", jobKey) // TODO: This log is a candidate for removal. It's now recorded in metadata
```

Comment:
Favor, only if you end up making other changes: Do you mind expanding `t` to `thowable`?